test/school_house_web/views/html_helpers_test.exs

```diff
@@ -0,0 +1,57 @@
+defmodule SchoolHouseWeb.HtmlHelpersTest do
```

Comment:
There's one more remaining function in the HtmlHelpers module that needs test coverage. I'll take care of that later.